core/src/main/java/io/opencensus/stats/Measure.java

```diff
@@ -31,7 +35,10 @@
       Function<? super MeasureLong, T> p1);
 
   /**
-   * Name of measure, as a {@code String}.
+   * Name of measure, as a {@code String}. Should be a ASCII string with a length no greater than
+   * 255 characters.
+   *
+   * <p>Suggested format for name: {@code <web_host>/<path>}.
```

Comment:
I think this is following some RPC naming conventions but I'm not sure either. @dinooliva can you answer this question?